ifrs17/Import/Importers.ipynb

```diff
@@ -1189,8 +1191,32 @@
                 "\n                                            if ( Enum.TryParse(interpolationMethodInput, out InterpolationMethod ipm)) ",
                 "\n                                                interpolationMethod = ipm;",
                 "\n                                            else if ( !(periodicity == (CashFlowPeriodicity)default && string.IsNullOrEmpty(interpolationMethodInput)) ) { ApplicationMessage.Log(Error.InvalidInterpolationMethod, dataNode); return null; }",
-                "\n                                        }",
-                "\n",
+                "\n                                    }",
+                "\n",
+                "\n                                    string economicBasisDriver = default;",
+                "\n                                    if(hasEconomicBasisDriver)",
+                "\n                                    {",
+                "\n                                        var economicBasisDriverInput = datarow.Field<string>(nameof(SingleDataNodeParameter.EconomicBasisDriver));",
+                "\n                                        if (!string.IsNullOrEmpty(economicBasisDriverInput))",
+                "\n                                            economicBasisDriver = economicBasisDriverInput;",
+                "\n                                        else { ApplicationMessage.Log(Error.InvalidEconomicBasisDriver, dataNode); return null; }",
+                "\n                                    }",
+                "\n                                    ",
+                "\n                                    double[] amortizationFactors = default;",
+                "\n                                    if(hasAmortizationFactors)",
+                "\n                                    {",
+                "\n                                        var amortizationFactorsInput = datarow.Table.Columns",
+                "\n                                            .Where(c => c.ColumnName.StartsWith(nameof(SingleDataNodeParameter.AmortizationFactor)))",
+                "\n                                            .OrderBy(c => c.ColumnName.Length).ThenBy(c => c.ColumnName)",
+                "\n                                            .Select(x => datarow.Field<string>(x.ColumnName).CheckStringForExponentialAndConvertToDouble())",
+                "\n                                            .ToArray()",
+                "\n                                            .Prune();",
+                "\n",
+                "\n                                        if (amortizationFactorsInput.Any())",
```

Comment:
This logs an error if the amortization factors are missing (even if the column are present). 
I think this case should be allowed. 
Consider a file with PAA GIC (with patterns) and a BBA GIC (without patterns because it has CU in the cashflow)

Reply:
can not find this "amortizationFactorsInput" anymore